client/components/public/FAQ.jsx

```diff
@@ -243,9 +323,9 @@ FAQ = class FAQ extends Component {
             <span>Contact us to:</span>
             <List.Item
                 header="Create a Division*"
-                description="Examples: Family Division or Club (Dance, Running, Book, Garden, Wine, ...) where team members are family or in that club."
+                description="Examples: Family Division, COVID Pod, or Club (Dance, Running, Book, Garden, Wine, &#x2026;) where team members are family or in that club."
```

Comment:
can you make this `&hellip;` like above?

Reply:
Fixed!